presto-orc/src/main/java/com/facebook/presto/orc/OrcWriterOptions.java

```diff
@@ -354,6 +363,12 @@ public Builder withPreserveDirectEncodingStripeCount(int preserveDirectEncodingS
             return this;
         }
 
+        public Builder withFlattenedColumns(Set<Integer> flattenedColumns)
+        {
+            this.flattenedColumns = flattenedColumns;
```

Comment:
Do you want to create a copy of ImmutableSet. The caller can mutate the set, theoretically in this model.

Reply:
modify the toString method and its associated tests to include the new flattenedColumns.

Reply:
Oh, I forgot to add a getter 😁.

Do we even need a test for toString?

Reply:
We can delete the tests. But some times, it helps me caught issues from mindless copy/pasting in ignoreDictionary, I copy/pasted changed the method name, but I was returning the copied variable and not new variable.

Reply:
Done: added getter, updated test case, added test case for default values.